server/testutils/factories/prison.ts

```diff
@@ -2,23 +2,26 @@ import { faker } from '@faker-js/faker/locale/en_GB'
 import { Factory } from 'fishery'
 
 import prisonAddressFactory from './prisonAddress'
-import type { Prison } from '@prison-api'
+import type { Prison } from '@prison-register-api'
 
 export default Factory.define<Prison>(({ params }) => {
-  const locality = params.locality || faker.location.county()
-  const city = params.city || faker.location.city()
-  const country = params.country || 'United Kingdom'
-  const premise = params.premise || `HMP ${locality}`
+  const county = faker.location.county()
+  const prisonName = params.prisonName || `${county} (HMP)`
```

Comment:
Probably fine as a factory default, but if we're supporting YOIs as well, we could randomise this with the `arrayElement` helper again

Reply:
I think this is fine - if we do want to add YOIs we can set them manually.